istioctl/pkg/admin/istiodconfig.go

```diff
@@ -485,10 +485,15 @@ func istiodLogCmd(ctx cli.Context) *cobra.Command {
 			var podName, ns string
 			if len(args) == 0 {
 				// resolvedRevision is already set by ctx.RevisionOrDefault(opts.Revision)
+				// For non-revisioned installs, if resolvedRevision is empty, use "default"
+				revisionForSelector := resolvedRevision
```

Comment:
Does this bug exist for other commands? should `RevisionOrDefault` fallback to `default` instead of `""`?